scrapers/DollsHub.yml

```
@@ -0,0 +1,41 @@
+name: DollsHub
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - filthygapers.com
+      - gynoexclusive.com
+      - ihuntmycunt.com
+      - maturegapers.com
+      - maturegynoexam.com
+      - maturegynospy.com
+      - nastypublicsex.com
+      - oldsfuckdolls.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="col-sm-12 col-lg-12 col-md-12 scence-title-name"]/h2
+      Performers:
+        Name: //li/span/a
```

most performers seem to be single named
adding the URL makes it easier to distinguish them later 
```suggestion
        Name: //li/span/a
        URL:  //li/span/a/@href
```

---

All sorted , Good point in adding the url.